do-challenges.html

```diff
@@ -417,6 +437,48 @@ <h3 name="wind-whistle">Challenge: Whistle for the Wind</h3>
                 </ul>
                 </p>
 
+                <a name="to-the-moon"></a>
+                <h3>Challenge: To the Moon</h3>
+                <p>
+                  Write a game about flying to the moon. There should be three stages in your
+                  game. One set on Earth; one in space and one on the moon.
+                </p>
+                <p>
+                  To start the rocket the player should press the space bar five times. Then the
+                  rocket should fly into the air. When the rocket reaches the edge of the screen,
+                  the background should be changed to the next background (e.g. from Earth to space). Along with placing the
+                  rocket at the bottom of the screen again. The player should be
+                  able to move the rocket left and right using the arrow keys. There should be
+                  Aliens and other space-ships flying around, if the player touches one of these
+                  they lose the game. When the player reaches the last stage (the moon), a target
```

Comment:
When the rocket enters the final moon screen, is the moon (target) at the top or the bottom of the screen? Was wondering this, as the previous screens have the rocket going from the bottom to the top (lift off and travel though space)